src/jit/liveness.cpp

```diff
@@ -1977,6 +1977,45 @@ void Compiler::fgComputeLife(VARSET_TP&       life,
     }
 }
 
+static bool HasAnyThrowableNodes(Compiler* compiler, BasicBlock* block, jitstd::vector<GenTree*>& disabledNodes)
+{
+
+    LIR::Range& blockRange  = LIR::AsRange(block);
+    GenTree*    currentNode = blockRange.LastNode();
+    GenTree*    endNode     = blockRange.FirstNonPhiNode()->gtPrev;
+
+    size_t left = disabledNodes.size();
+
+    while (currentNode != endNode)
+    {
+        for (size_t i = 0; i < disabledNodes.size(); ++i)
```

Comment:
I understand that `disabledNodes.size()` is small, but I would prefer `set` or `hash map` here, but it is up to you.

Reply:
both set and hashmap consume much more memory. I think in case of small 'n' it's better to go with flat data structures.